.github/workflows/test.yml

```diff
@@ -329,25 +329,53 @@ jobs:
       - name: Test GTK Build
         run: nix develop -c zig build -Dapp-runtime=gtk -Dgtk-adwaita=true -Demit-docs
 
-      - name: Test GTK Build (No Libadwaita)
-        run: nix develop -c zig build -Dapp-runtime=gtk -Dgtk-adwaita=false -Demit-docs
-
-      - name: Test GTK build with explicit X11
-        run: nix develop -c zig build -Dapp-runtime=gtk -Dgtk-adwaita=true -Demit-docs -Dgtk-x11=true
-
-      - name: Test GTK build with no X11
-        run: nix develop -c zig build -Dapp-runtime=gtk -Dgtk-adwaita=true -Demit-docs -Dgtk-x11=false
-
-      - name: Test GTK build with no Adwaita and no X11
-        run: nix develop -c zig build -Dapp-runtime=gtk -Dgtk-adwaita=false -Demit-docs -Dgtk-x11=false
-
       - name: Test GLFW Build
         run: nix develop -c zig build -Dapp-runtime=glfw
 
       # This relies on the cache being populated by the commands above.
       - name: Test System Build
         run: nix develop -c zig build --system ${ZIG_GLOBAL_CACHE_DIR}/p
 
+  test-gtk:
+    strategy:
+      fail-fast: false
+      matrix:
+        adwaita: ["true", "false"]
+        x11: ["true", "false"]
+    runs-on: namespace-profile-ghostty-md
```

Comment:
I would suggest using enum like strings. Otherwise you won't know which job failed because it will look like `test-gtk(true, false)` or something

Reply:
I just named them using the `name:` key. Is there some other way? That's what I'm used to doing.

Reply:
It is just a UX problem in my opinion of GitHub that have to be worked around. No biggie